docs/source/admin/traffic_router.rst

```diff
@@ -601,16 +601,16 @@ Pattern Based Consistent Hashing Feature
 
 Overview
 --------
-Pattern based consistent hashing is a feature to modify the request path given to Traffic Router's consistent hasher for Cache selection (and Delivery Service selection for Steering Delivery Services) using a regex. This new regex field 'Consistent Hash Regex' is applied on a per-Delivery Service basis and is given to Traffic Router via the CRConfig. The purpose of this feature is to increase cache efficiency by directing requests for the same asset, but with varying request paths, to the same cache.
+Pattern-Based Consistent Hashing is a feature to modify the request path given to Traffic Router's consistent hasher for Cache selection (and :term:`Delivery Service` selection for Steering Delivery Services) using a regular expression. This new regular expression field 'Consistent Hash Regex' is applied on a per-Delivery Service basis and is given to Traffic Router via the CRConfig. The purpose of this feature is to increase cache efficiency by directing requests for the same asset, but with varying request paths, to the same Cache.
```

Comment:
It's preferable to always link a term to its definition, but once per section ("page" in an HTML context) is probably enough at a minimum.